config/core/configmaps/defaults.yaml

```diff
@@ -81,3 +81,11 @@ data:
     # disables this throttling and lets through as many requests as
     # the pod receives.
     container-concurrency: "0"
+
+    # The container concurrency max limit is an operator setting ensuring that
+    # the individual revisions cannot have arbitrary large concurrency
+    # values, or autoscaling targets. `container-concurrency` default setting
+    # must be at or below this value.
+    # If the value is `0`, no limit is enforced.
```

Comment:
This is not true, you fail if it is less than 1.  I think the default is 1000?

Reply:
The line below is correct

Reply:
I removed the "0" as a possible value for now.